test/contact.html

```diff
@@ -0,0 +1,107 @@
+<html>
+<head>
+ <!-- Latest compiled and minified CSS -->
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/css/bootstrap.min.css">
+<style>
+.form-control-feedback {
+position: relative;
+display: inline;
+top: 0;
+line-height: 14px;
+}
+</style>
+
+<!-- Optional theme -->
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/css/bootstrap-theme.min.css">
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
+<!-- Latest compiled and minified JavaScript -->
+<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/js/bootstrap.min.js"></script>
+<script src="https://cdn.firebase.com/js/client/2.2.3/firebase.js"></script>
+
+<script>
+    function getUrlVars() {
+        var vars = {};
+        var parts = window.location.href.replace(/[?&]+([^=&]+)=([^&]*)/gi, function(m,key,value) {
+            vars[key] = value;
+        });
+        return vars;
+    }
+
+    var onSubmitComplete = function(error) {
+        var contactForm = document.getElementById('contact-form');
+        var contactResponse = document.getElementById('contact-response');
+        var contactBtn = document.getElementById('btn-submit');
+      contactBtn.disabled = false;
+      if (error) {
+        contactResponse.innerHTML = 
+            '<div class="alert alert-danger">Sorry. Could not submit the error report.</div>';
+      } else {
+        contactResponse.innerHTML =
+            '<div class="alert alert-success">Thanks for submitting your error report!</div>';
+        // hide the form
+        contactForm.style.display = 'none';
+      }
+    };
+
+    function reporterror(formObj) {
+        // Store error information to firebase
+        var contactBtn = document.getElementById('btn-submit');
+        var myFirebaseRef = new Firebase("https://epiclogger.firebaseio.com/errors");
+        var id = getUrlVars()['id'];
+        var currentdate = new Date(); 
+        var datetime = currentdate.getDate() + "/"
+                + (currentdate.getMonth()+1)  + "/" 
+                + currentdate.getFullYear() + " @ "  
+                + currentdate.getHours() + ":"  
+                + currentdate.getMinutes() + ":" 
+                + currentdate.getSeconds();
+        myFirebaseRef.push({
+          'id' : id,
+          'email' : formObj.InputEmail.value,
+          'notes' : formObj.InputMessage.value,
+          'timestamp': datetime
+        }, onSubmitComplete);   
+        contactBtn.disabled = true;
+        return false;
+    }
```

Comment:
I think we should move this part of code and update the loader script. This code should be integrated so we don't load have js in 2 places even though i understand that it's for submitting the data and that's why you did it but what i had in mind was that we should have even the html and css part in the script so we don't load an iframe any more. 

So the loading will be super fast and we don't have to host this page anywhere on any server to load it. Seems fair?